src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -765,6 +767,9 @@ def clone_subvolume_snapshot(self, **kwargs):
         s_groupname  = kwargs['group_name']
 
         try:
+            if get_all_pending_clones_count(self, self.mgr, self.volspec) >= self.mgr.max_concurrent_clones:
+                raise(VolumeException(-errno.EAGAIN, "clone creation request exceeds maximum limit"))
```

Comment:
Could we improve error messsage here ? How about "all cloner threads are busy, please try again later" ?

Reply:
Updated the error message.